addons/mail/static/src/models/autocomplete-input-view/autocomplete-input-view.js

```diff
@@ -0,0 +1,295 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one2one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'AutocompleteInputView',
+    identifyingFields: [['chatWindowOwnerAsNewMessageFormInput', 'discussAsMobileAddItemHeader', 'discussSidebarCategoryAsAddingItemInputView']],
+    recordMethods: {
+        doFocus() {
+            if (!this.component) {
+                return;
+            }
+            this.component.root.el.focus();
+        },
+        /**
+         * @param {MouseEvent} ev
+         */
+        onBlur(ev) {
+            this._hide();
+        },
+        onFocusin() {
+            this.update({ isFocused: true });
+        },
```

Comment:
Those 2 handlers look dangerous being unguarded with exists(), I know blur was an issue in another PR.